bin/aisdecode

```diff
@@ -37,7 +50,7 @@ for key in ("uscg", "validate_checksum", "verbose", "allow_unknown", "treat_ab_e
         args[key] = args[key] == 'True'
 
 if "help" in args:
-    print """Usage: aisdecode [OPTIONS] < INFILE.nmea > OUTFILE.json
+    print("""Usage: aisdecode [OPTIONS] < INFILE.nmea > OUTFILE.json
```

Comment:
@schwehr No - a lot of the functionality in `__future__` was backported.  This works for `2.6` as well.